mainStyle.css

```diff
@@ -0,0 +1,203 @@
+body{
+    max-height: 1800px;
+    color:white;
+    font-family: 'Montserrat', -apple-system, BlinkMacSystemFont, 'Myriad Pro', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+}
+
+*{
+    margin: 0px;
+    padding: 0px;
+}
+
+a:link {
+    color: white;
+}
+  
+a:visited {
+    color: white;
+}
+  
+a:hover {
+    color: white;
+}
+  
+a:active {
+    color: white;
+  }
+
+textarea:focus, input:focus{
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs

Reply:
replaced tag selectors w/ classes.